commands/index.ts

```diff
@@ -60,20 +72,22 @@ export class Commands {
     return this.commandMap.get(command.trim())
   }
   
-  public async execute(message: string): Promise<boolean> {
-    const [command, query]: (string | undefined)[] = message.trim().split(SPLIT_OPERAND_QUERY_REGEX).slice(1)
+  public async execute(message: string): Promise<CommandResponse> {
+    const split: (string | undefined)[] = message.trim().split(/\s+/)
+    const command = split[0]
```

Comment:
Seems a bit odd, thought about something like `const [command, query] = (split => ... || [])(message.trim().split(/\s+/)` but doesn't seem more readable neither.

Reply:
Yeah, I want to revisit this.